tests/integration/test_wpk/test_wpk_manager_task_states.py

```diff
@@ -1,7 +1,75 @@
-# Copyright (C) 2015-2021, Wazuh Inc.
-# Created by Wazuh, Inc. <info@wazuh.com>.
-# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
-
+'''
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Register agents in the manager and test upgrading
+        them through a command, which will make the agent
+        to upgrade using WPK packages.
```

Comment:
```suggestion
brief: Agents can be upgraded remotely. This upgrade is performed by the manager which sends each registered agent a WPK (Wazuh signed package) file that contains the files needed to upgrade the agent to the new version. These tests ensure, the behaviour of the WPK upgrade on the manager side, in case of the manager stopped before finishing the upgrade.
```

Reply:
Done in this [commit](https://github.com/wazuh/wazuh-qa/pull/2409/commits/210acc6c36122300661af5bbcefdc5b6d158bbb1)